src/index.js

```diff
@@ -177,9 +177,12 @@ function doWatch(options, cwd, steps) {
 					}
 				}
 				if (e.code === 'END') {
-					options._sizeInfo.then(text => {
-						stdout(`Wrote ${text.trim()}`);
-					});
+					// fix NPE
+					if (options._sizeInfo) {
+						options._sizeInfo.then(text => {
+							stdout(`Wrote ${text.trim()}`);
+						});
+					}
```

Comment:
```suggestion
					if (options._sizeInfo) {
						options._sizeInfo.then(text => {
							stdout(`Wrote ${text.trim()}`);
						});
					}
```

nit: An undefined check is obvious enough to not require a comment.